templates/typescript-mcp/README.md

```
@@ -1,14 +1,20 @@
 # TypeScript MCP Template
 
-This template provides a complete example of building AI-powered applications with MooseStack and the Model Context Protocol (MCP).
+This template provides a complete example of building AI-powered chat-over-data applications with MooseStack and the Model Context Protocol (MCP).
 
 ## Overview
 
 This is a pnpm monorepo containing two independent applications that work together:
 
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)